tests/cmdline/commands/test_storage.py

```diff
@@ -111,7 +111,10 @@ def tests_storage_maintain_logging(run_cli_command, monkeypatch, caplog):
 
     from aiida.backends import control
 
-    def mock_maintain(**kwargs):
+    def mock_maintain(*args, **kwargs):
+        logmsg = 'Provided args:\n'
+        for argmnt in args:
```

Comment:
Why not go all the way and do `rgmnt` if we are getting rid of vowels :sweat_smile: Honestly, I would really just use the typical `arg` or `argument` here.